openreview/venue_request/venue_request.py

```diff
@@ -1179,7 +1179,7 @@ def setup_request_form(self):
             },            
             'Submission Start Date': {
                 'description': 'When would you (ideally) like to have your OpenReview submission portal opened? Please specify the date and time in GMT using the following format: YYYY/MM/DD HH:MM(e.g. 2019/01/31 23:59). (Skip this if only requesting paper matching service)',
```

Comment:
Perhaps we should change the end of the description a little bit to make it more clear:

> Leave blank if you would like the portal to open for submissions as soon as possible or if you are only requesting paper matching service.